pynicotine/gtkgui/search.py

```diff
@@ -1587,20 +1650,21 @@ def on_refilter(self, *_args):
         # Replace generic file type filters with real file extensions
         file_type_filters = filters["filtertype"]
 
-        for filter_name, file_extensions in self.FILTER_GENERIC_FILE_TYPES:
-            excluded_filter_name = f"!{filter_name}"
```

Comment:
It might be best for this code to exist in its own separate function(s).

Reply:
Not addressing this right now.